test/controllers/tasks_controller_test.rb

```diff
@@ -124,10 +119,14 @@ def in_taxonomy_scope(organization, location = nil)
           assert_include response.body.lines[1], 'Child action'
         end
 
-        it 'shows duration column in exported task' do
-          task = ForemanTasks::Task.with_duration.find(FactoryBot.create(:some_task).id)
-          get(:sub_tasks, params: { id: task.id, duration: task.duration }, session: set_session_user)
+        it 'shows duration column' do
+          parent = ForemanTasks::Task.with_duration.find(FactoryBot.create(:some_task).id)
```

Comment:
The only thing we need to know about the parent task is the id, so we should be fine with just `FactoryBot.create(:some_task).id`. There's no need to reload it with duration